rss2email/__init__.py

```diff
@@ -49,5 +49,5 @@
 
 if _sys.version_info < (3, 2):
     raise ImportError(
-        "rss2email requires Python 3.2, but you're using:\n{}".format(
-            _sys.version))
+        "rss2email requires Python 3.2 or older, but you're
```

Comment:
rss2email is incompatible with versions of Python older than 3.2. It requires Python 3.2 or *newer*